dev-haskell/beam-migrate/beam-migrate-0.5.0.0.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour"
+inherit haskell-cabal
+
+DESCRIPTION="SQL DDL support and migrations support library for Beam"
+HOMEPAGE="https://travis.athougies.net/projects/beam.html"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE="werror"
```

Comment:
Same comment, I don't think we need the `werror` USE-flag. I think there's a way to tell hackport to leave this out, but I'm not really sure how to do that. pinging @solpeth

Reply:
`hackport merge` command has a `-f` flag to force flag values and remember them for next merges. `hackport merge -f-werror beam-migrate` should do the right thing.